pages/admin/organisationen/KlasseManagementView.neu.page.ts

```diff
@@ -0,0 +1,87 @@
+import { expect, type Locator, Page } from '@playwright/test';
+import { Autocomplete } from '../../../elements/Autocomplete';
+import { DataTable } from '../../components/DataTable.neu.page';
+import { KlasseDetailsViewPage } from './KlasseDetailsView.neu.page';
+import { AbstractManagementViewPage } from '../../abstracts/AbstractManagementView.page';
+import { SearchFilter } from '../../../elements/SearchFilter';
+
+export class KlasseManagementViewPage extends AbstractManagementViewPage {
+  /* add global locators here */
+  private readonly klasseTable: DataTable = new DataTable(this.page, this.page.getByTestId('klasse-table'));
+  private readonly searchFilter: SearchFilter = new SearchFilter(this.page);
+
+  constructor(page: Page) {
+    super(page);
+  }
+
+  /* actions */
+  public async waitForPageLoad(): Promise<void> {
+    await this.page.getByTestId('klasse-management-card').waitFor({ state: 'visible' });
+    await expect(this.page.getByTestId('layout-card-headline')).toHaveText('Klassenverwaltung');
+    return expect(this.page.getByTestId('klasse-table')).not.toContainText('Keine Daten');
+  }
+
+  public async filterBySchule(schulname: string): Promise<void> {
+    const schuleFilter: Autocomplete = new Autocomplete(this.page, this.page.getByTestId('schule-select'));
+
+    await schuleFilter.searchByTitle(schulname, false);
+  }
+
+  public async filterByKlasse(klassenname: string): Promise<void> {
+    const klasseFilter: Autocomplete = new Autocomplete(this.page, this.page.getByTestId('klasse-select'));
+
+    await klasseFilter.searchByTitle(klassenname, false);
+  }
```

Comment:
```suggestion
private async filterBy(testId: string, name: string): Promise<void> {
  const filter: Autocomplete  = new Autocomplete(this.page, this.page.getByTestId(testId));
  await filter.searchByTitle(name, false);
}

public async filterBySchule(schulname: string): Promise<void> {
  await this.filterBy('schule-select', schulname);
}

public async filterByKlasse(klassenname: string): Promise<void> {
  await this.filterBy('klasse-select', klassenname);
}
```

We can dry up the  creation of the filter.